lib/core.sh

```diff
@@ -396,13 +401,10 @@ function build_image_env(){
     sudo bash -c "echo 'export PATH=/opt/yaourt/bin:\$PATH' > ${maindir}/root/etc/profile.d/${CMD}.sh"
     sudo chmod +x ${maindir}/root/etc/profile.d/${CMD}.sh
 
-    info "Install ${NAME} script..."
-    sudo pacman --noconfirm --root ${maindir}/root -S git
-    _install_from_aur ${maindir} "${CMD}-git" "${CMD}.install"
-    sudo pacman --noconfirm --root ${maindir}/root -Rsn git
-
     info "Setting up the pacman keyring (this might take a while!)..."
-    sudo arch-chroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux"
+    sudo pacman --noconfirm --root ${maindir}/root -S psmisc
+    sudo ${maindir}/root/opt/junest/bin/jchroot ${maindir}/root bash -c "pacman-key --init; pacman-key --populate archlinux; killall gpg-agent"
```

Comment:
If (for any bizarre reason) a user is running a second junest sandbox and is doing something with gpg-agent there, then it will be killed as well?.. I guess we should be more discreet.

There is a command `pgrep -P $$`, which will print PID of all processes whose PPID is current process PID. Then they can be killed with `pgrep .. | xargs kill`. To go through grandchildren it has to be called again with children's PID. I could not find any simple way to print find all children's grandchildrens and so on, but we could probably cope with a simple recursive function like this:

```
get_children() {
    local ppid=$1
    local pid
    for pid in $(pgrep -P ${ppid})
    do
        get_children ${pid}
        echo ${pid}
    done
}
```

Then it can be made available inside junest by `export -f get_children` and its output fed through xargs to kill.